distributions/Jamulus.1

```diff
@@ -277,9 +276,9 @@ online handbook
 .It Pa https://jamulus.io/wiki/FAQ
 frequently asked questions
 .It Pa https://jamulus.io/wiki/Running\-a\-Server
-documentation on server configuration and types
-.It Pa https://jamulus.io/wiki/Server\-Linux#running\-in\-public\-mode
-current list of directory servers operated by the Jamulus project,
+documentation on Server configuration and types
+.It Pa https://jamulus.io/wiki/Server\-Linux#running\-in\-registered\-mode
```

Comment:
404 Not Found

Don’t break the hyperlink.

Reply:
It won't get released until the website is updated.